test/key_value_store.js

```diff
@@ -511,6 +511,8 @@ describe('KeyValueStore', () => {
 
             delete process.env[ENV_VARS.LOCAL_STORAGE_DIR];
             delete process.env[ENV_VARS.INPUT_KEY];
+
+            defaultStore.getValue = oldGet.bind(defaultStore);
```

Comment:
Just a note. No need to `.bind` here. `this` will be `defaultStore` automatically, because it's the calling object `defaultStore.getValue`.